apps/src/sites/studio/pages/blocks/edit.js

```diff
@@ -20,6 +20,18 @@ let poolField, nameField, helperEditor, configEditor, validationDiv;
 
 $(document).ready(() => {
   registerReducers({animationList: animationList});
+  getDefaultListMetadata()
+    .then(defaultSprites => {
+      initializeEditPage(defaultSprites);
+    })
```

Comment:
i alluded to this in [this comment](https://github.com/code-dot-org/code-dot-org/pull/43615/files#r751674696), but there are some ES6 features we can use to simplify arrow function callbacks like this:

```javascript
// single-line statements don't require curly braces
getDefaultListMetadata().then(defaultSprites => initializeEditPage(defaultSprites));

// if a function's signature matches a callback's signature, you can use the shorthand below.
// in this case, the signature of initializeEditPage matches the signature of our .then callback
getDefaultListMetadata().then(initializeEditPage);
```

i think it's important to call out because we use these features a lot in react class components:
```javascript
class MyComponent extends React.Component {
  onClickButton = event => {
    // do something with click event
  }

  render() {
    // the two buttons below are equivalent and use both features expressed above
    return <div>
      <button onClick={e => this.onClickButton(e)}>hi</button>
      <button onClick={this.onClickButton}>hi</button>
    </div>
  }
}
```

you just have to be careful with scoping -- [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) are scoped differently than [function declarations](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function), so you mainly have to make sure `this` is what you think it is when you use these features.